Resources/Storage/GFAL2_StorageBase.py

```diff
@@ -57,18 +57,12 @@ def __init__( self, storageName, parameters ):
     # # gfal2 API
     self.gfal2 = gfal2.creat_context()
     self.gfal2.set_opt_boolean( "BDII", "ENABLE", False )
-    # # save c'tor params
-    self.name = storageName
-    self.protocol = parameters['Protocol']
     self.spaceToken = parameters['SpaceToken']
-
-    # #stage limit - 12h
```

Comment:
I thought they didn't add much. The variables are called stageTimeout and gfal2Timeout (and one comment just said gfal2 timeout again).
I put them back in if that makes it more readable.